packages/identifier/tests/resolve.test.ts

```diff
@@ -48,12 +48,12 @@ describe('handle resolution', () => {
   })
 
   it('handles a bad DNS resolution', async () => {
-    await expect(resolveDns('bad.test')).rejects.toThrow(NoHandleRecordError)
+    const did = resolveDns('bad.test')
```

Comment:
should this be an await statement? it's failing tests

Reply:
yup 👍